packages/react/src/bags/hooks/useBagItem.ts

```diff
@@ -431,13 +333,7 @@ const useBagItem: UseBagItem = bagItemId => {
      * @type {boolean}
      */
     isLoading,
-    /**
-     * @type {Function}
-     * @variation Member
-     *
-     * @see {@link module:useBagItem~handleAddOrUpdateItem|handleAddOrUpdateItem} method
-     */
-    handleAddOrUpdateItem,
```

Comment:
Wild idea: what if we still export `handleAddOrUpdateItem` with the bag item already provided, like `handleAddOrUpdateItem: handleAddOrUpdateItem(bagItem)`? Otherwise, the developer would probably end up importing both hooks 99% of the time it imports `useBagItem`, or am I missing something? 🤔

Reply:
This approach would call `handleAddOrUpdateItem` twice, so it would need to be tweaked. 

Either way, I chose not to go forward with it, because in our use case scenarios we never have to call both hooks at the same time (mainly because, we will only use `useAddOrUpdateItem` to add and never to update, since `useBagItem` already handles the updates we need). The cases you are referring to happen only when we want to update the bag item and honestly I don't see a problem with that (in fact, it's exactly what is suggested in #16 )